src/interfaces/IMetaMorpho.sol

```diff
@@ -70,7 +70,8 @@ interface IMetaMorphoBase {
     function withdrawQueueLength() external view returns (uint256);
 
     /// @notice Stores the total assets managed by this vault when the fee was last accrued.
-    /// @dev May be a little off `totalAssets()` after each interaction, due to some roundings.
+    /// @dev May be greater than `totalAssets()` between deposits/withdrawals due to roundings or socialized bad debt.
```

Comment:
```suggestion
    /// @dev May be greater than `totalAssets()` between deposits/withdrawals due to forced market removal or socialized bad debt.
```
I know this is specified below but it feels like this list should be exhaustive so I believe forced market removal should be added here also.
Also what kind of rounding would make it be smaller ? If the same computation is used everytime, and since division is increasing in the numerator, I think that rounding errors are not the culprit here

Reply:
The balance is rounded down in totalAssets() so after a deposit, `lastTotalAssets` may be n assets greater where n is the nb of markets deposited to
Same upon withdrawal

Reply:
I would phrase it "due to removal of markets with non-zero supply" instead of "due to forced market removal"